hypnos/src/clock.c

```diff
@@ -59,6 +62,12 @@ struct tm *clock_get_time()
 	return &ti;
 }
 
+void clock_sync_time(void)
+{
+	cts_get_datetime(&ti);
```

Comment:
Same thing here. I know that "ti" is from an old implementation, but i realize now that it is hard to understand what "ti" is.

Reply:
Well it can't be called `time` and I want to have a different name for static and local variables.